docs/blog/announcing-vite5-1.md

```diff
@@ -0,0 +1,97 @@
+
+The preview server now exposes a `close` method. See [#15630](https://github.com/vitejs/vite/issues/15630).
+
+## Performance improvements
+
+Vite keeps getting faster with each release, and Vite 5.1 is packed with performance improvements.
+
+Vite now has opt-in support for running CSS preprocessors in threads. You can enable it using [`css.preprocessorMaxWorkers: true`](https://main.vitejs.dev/config/shared-options.html#css-preprocessormaxworkers). For a Vuetify 2 project, dev startup time was reduced by 40% with this feature enabled. There is [performance comparison for others setups in the PR](https://github.com/vitejs/vite/pull/13584#issuecomment-1678827918). See [#13584](https://github.com/vitejs/vite/issues/13584).
+
+There are also new tools to speed up dev server cold start. You can set `optimizeDeps.holdUntilCrawlEnd: false` to switch to a new strategy for deps optimization that may help in big projects. We're considering switching to this strategy by default in the future. See [#15244](https://github.com/vitejs/vite/issues/15244).
+
+The `fs.cachedChecks` optimization is now enabled by default. In Windows, `tryFsResolve` was ~14x faster with it, and resolving ids overall got a ~5x speed up in the triangle benchmark. See [#15704](https://github.com/vitejs/vite/issues/15704).
+
+The dev server had several incremental performance gains. A new middleware to short-circuit on 304 ([#15586](https://github.com/vitejs/vite/issues/15586)). We avoided `parseRequest` in hot paths ([#15617](https://github.com/vitejs/vite/issues/15617)). Rollup wasn't properly lazy loaded ([#15621](https://github.com/vitejs/vite/issues/15621))
+
+## Deprecations
+
+We continue to reduce Vite's API surface where possible to make the project manintainable long term.
+
+The `as` option to `import.meta.glob` has been deprecated. The standard moved to [Import Attributes](https://github.com/tc39/proposal-import-attributes), but we don't plan to replace `as` with a new option at this point. Instead, it is recommended that the user switches to `query`. See [#14420](https://github.com/vitejs/vite/issues/14420).
+
+Build time pre-bundling, an experimental feature added in Vite 3, was removed. With Rollup 4 switching its parser to native, and Rolldown being worked on, both the performance and the dev-vs-build inconsistency story for this feature are no longer valid. If we want to focus on improving dev/build consistency, then using Rolldown for prebundling during dev and Rolldown as-is during build is a better bet moving forward. Rolldown may also implement caching in a way that is a lot more efficient during build than deps prebundling. See [#15184](https://github.com/vitejs/vite/issues/15184).
```

Comment:
I think it would be nice to mention that Rolldown was delayed to Q1-2 of this year since I'm pretty sure lots of users are curious about this subject. For reference I've asked this exact question in the Vite to Rust [discussion](https://github.com/vitejs/vite/discussions/12529) and got an answer directly from Evan in which he referenced his X post [An update on Rolldown](https://twitter.com/youyuxi/status/1741065178597536191)

So perhaps add this X link as an appendix!?

Reply:
I prefer to answer this one in social if it appears. I don't know if we should give another estimate in this blog post. Probably better to let the rolldown team announce that themselves as they keep progressing.